pkgs/development/python-modules/robotframework-selenium2library/default.nix

```diff
@@ -1,29 +1,22 @@
-{ stdenv
-, buildPythonPackage
-, fetchPypi
-, robotframework
-, selenium
-, docutils
-, decorator
-}:
+{ stdenv, buildPythonPackage, fetchPypi, robotframework-seleniumlibrary }:
 
 buildPythonPackage rec {
-  version = "1.6.0";
+  version = "3.0.0";
   pname = "robotframework-selenium2library";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "1asdwrpb4s7q08bx641yrh3yicgba14n3hxmsqs58mqf86ignwly";
+    sha256 = "2a8e942b0788b16ded253039008b34d2b46199283461b294f0f41a579c70fda7";
   };
 
-  # error: invalid command 'test'
-  #doCheck = false;
+  # no tests included in PyPI tarball
```

Comment:
I'm not even seeing tests on github.

Reply:
That's probably because with 3.0.0 `robotframework-selenium2library` [has become](https://github.com/robotframework/Selenium2Library/blob/b4af8d0015727d05831aa7a397b78d01c142ac4e/src/Selenium2Library/__init__.py#L38)
> just a thin wrapper to SeleniumLibrary that eases with transitioning to the new project